samples/nrf5340/smp_svr/README.rst

```diff
@@ -0,0 +1,168 @@
+.. _smp_svr_ext_xip:
+
+SMP Server Sample with external XIP
+###################################
+
+Overview
+********
+
+This sample implements a Simple Management Protocol (SMP) server that demonstrates splitting the application that it partially resides on internal flash and partially on QSPI flash.
+
+SMP is a basic transfer encoding for use with the MCUmgr management protocol.
+For more information about MCUmgr and SMP, please see :ref:`device_mgmt`.
+
+Refer to zephyr-rtos :zephyr:code-sample:`smp-svr` for information about SMP features implemented by the sample.
+
+Thes samples enables the spliting the application into intenal and external part thasnks to:
+
+* Enabled by :kconfig:option:`CONFIG_XIP_SPLIT_IMAGE` Split XIP image feature, which turns on the build system level support for image division and adapts image signing support for MCUboot.
+* Project CMake which describes relocation of part of libraries or object to the external XIP area.
+* Linker scripts file which describes external QSPI area to which code should be linked.
```

Comment:
"script file describing" I think sounds better than "which describes".
I think what the script does is it defines SoC RAM range reserved for mapping of external memory device, connected to the QSPI peripheral of SoC, which in turn allows you to build a code that is relocated to that mapped range.